### ISSUE TITLE
Plugin does not support Substitutions

### ISSUE THREAD

**Issue by DanielWenz** [Pretius/apex-helptext-tooltip#1]
Nice plugin, unfortunately it does not support substitution variables inside the Item help text.
Example: Variable (P4010_HELP_TEXT)  inside the item help text is displayed on thepage as &P4010_HELP_TEXT. and not with the text.

Are you planing to support this in the future?

**Comment by bostrowski** [comment]
@DanielWenz 
Yes, I can include it in next release which will contain fixes :-)

**Comment by DanielWenz** [comment]
That are great news!
